turbinia/workers/bulk_extractor.py

```diff
@@ -33,14 +33,17 @@ def run(self, evidence, result):
     Returns:
         TurbiniaTaskResult object.
     """
+    # TODO(wyassine): Research whether bulk extractor has an option to
+    # generate a summary report to stdout so that it could be used for
+    # a report in this task.
     # Create the new Evidence object that will be generated by this Task.
     output_evidence = CompressedDirectory()
     # Create a path that we can write the new file to.
     base_name = os.path.basename(evidence.local_path)
-    output_file_path = os.path.join(self.output_dir, '{0:s}'.format(base_name))
+    output_file_path = os.path.join(self.output_dir, base_name)
     # Add the output path to the evidence so we can automatically save it
     # later.
-    output_evidence.local_path = output_file_path
+    output_evidence.source = output_file_path
```

Comment:
Is this supposed to be `source_path`?

Reply:
In the `Evidence` object init, I see a variable definition for source with the following description `source (str): String indicating where evidence came from (including tool version that created it, if appropriate).`. I can initialize a new variable named `source_path` if that makes more sense though.

Reply:
Per chat with Aaron this should be OK.